### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by LukeWattsCP**
First wave of j unit tests, need to decide how to handle functions with id parameters and if we need more in depth for the other functions. Currently ensuring non null values are returned and that function performs.

### CODE REVIEW

File: unit-tests/song-services.test.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    expect(result).toBeTruthy();
+});
+
+
+test('Testing findSongByTitle()', () => {
+    const result = songServiceFunctions.findSongByTitle(testSongTitle)
+    expect(result).toBeTruthy();
+});
+
+
```
Comment (heberman) [inline unit-tests/song-services.test.js:47]: Maybe add test(s) for when playlist being searched is not in the db?

File: unit-tests/song-services.test.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+var testSongTitle = 'Easy On Me'
+
+var testPlaylist = {
+    title: "Test Playlist",
+    songs: [{
+        title: 'Easy On Me',
+        Artist: 'Adele',
+        runtime: 3.45,
+    }]
```
Comment (ReedMcCullough) [inline unit-tests/song-services.test.js:13]: When testing the usage of playlists, does the number of songs in the playlist matter? If not, good job only using one song

File: unit-tests/song-services.test.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+//     const result = null
+    
+//     expect(result).toBeTruthy();
+// });
+
+// test('Testing editPlaylist()', () => {
+//     const result = null
+    
+//     expect(result).toBeTruthy();
+// });
```
Comment (ReedMcCullough) [inline unit-tests/song-services.test.js:64]: Do we not use these functions anymore? Or is there another reason we don't use these tests...

File: unit-tests/song-services.test.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+test('Testing addPlaylist()', () => {
+    const result = songServiceFunctions.addPlaylist(testPlaylistJson);
+    expect(result).toBeTruthy();
+});
+
+
+test('Testing findPlaylistByTitle()', () => {
+    const result = songServiceFunctions.findPlaylistByTitle(testPlayListTitle)
+    expect(result).toBeTruthy();
+});
```
Comment (ReedMcCullough) [inline unit-tests/song-services.test.js:27]: Even though we retrieve information, its interesting to see we can just use boolean values to test our code still

**Pull request closed without merging**